portfolio/src/main/webapp/index.html

```diff
@@ -12,10 +12,15 @@
     <div id="content">
       <h1>Catalina's Portfolio</h1>
       <img class ="avatar" src="images/myAvatar.jpg"/>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random fact about me:</p>
+      <p>Click here to get a greeting:</p>
+      <button onclick="showServletMess()">Message</button>
```

Comment:
Nit: I would suggest changing the name of this function to showServletMessage(). The resulting function name isn't that long, and it highly increases readability. I had to think for a second to understand what this function name meant (especially since mess is an actual word), while `showServletMessage()` is immediately clear. We want to prioritize readability over trying to shorten function names as much as possible.